### PR TITLE
feature: Add SSH button in the agent access column

### LINKED ISSUE #2881: Clarify that SSH access is possible in dashboard
As a user I may look at this:

Screen Shot 2022-07-09 at 3 48 36 PM

And assume that there is no way to use my local SSH tools with Coder.

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Closes #2881 

Screen Shot 2022-07-12 at 11 15 56

### CODE REVIEW

File: site/src/components/Resources/Resources.tsx [hunk 1]
```diff
@@ -107,18 +108,17 @@ export const Resources: FC<ResourcesProps> = ({
                         <span style={{ color: agentStatus.color }}>{agentStatus.status}</span>
                       </div>
                     </TableCell>
-                    {canUpdateWorkspace && (
-                      <TableCell>
-                        <div className={styles.accessLinks}>
-                          {agent.status === "connected" && (
+                    <TableCell>
+                      <>
+                        {canUpdateWorkspace && agent.status === "connected" && (
```
Comment (presleyp) [inline site/src/components/Resources/Resources.tsx:113]: Thanks for cleaning up the checks

File: site/src/components/SSHButton/SSHButton.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          vertical: "bottom",
+          horizontal: "left",
+        }}
+        transformOrigin={{
+          vertical: "top",
+          horizontal: "left",
+        }}
+      >
+        <HelpTooltipText>
+          To connect with SSH you have to run the following commands:
```
Comment (presleyp) [inline site/src/components/SSHButton/SSHButton.tsx:58]:
```suggestion
          Run the following commands to connect with SSH:
```
I think this is a little more clear and direct

**Review by mafredri — COMMENTED** [review]
This looks really nice!

I noticed we're not mentioning the option to SSH via coder CLI (`coder ssh`) but since it's a core CLI feature, maybe it doesn't need mention.

File: site/src/components/SSHButton/SSHButton.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (presleyp) [reply site/src/components/SSHButton/SSHButton.tsx:58]: Looks like we were thinking similarly. The note about when to run `config-ssh` could be added to the label "Configure ssh" (currently line 64)

File: site/src/components/SSHButton/SSHButton.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+          <div>
+            <HelpTooltipText>
+              <strong className={styles.codeExampleLabel}>Connect to the agent</strong>
+            </HelpTooltipText>
+            <CodeExample code={`ssh coder.${workspaceName}.${agentName}`} />
+          </div>
+        </Stack>
+
+        <HelpTooltipLinksGroup>
```
Comment (presleyp) [inline site/src/components/SSHButton/SSHButton.tsx:77]: What do you think about having some text before these links introducing them as docs? Like "For more information:" or something like that.

**Review by presleyp — APPROVED** [review]
Good to go once the wording is all settled, imo

File: site/src/components/SSHButton/SSHButton.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (mafredri) [reply site/src/components/SSHButton/SSHButton.tsx:58]: Haha, indeed. I think your proposal is a tad better. And moving the config-ssh sounds good!

[the remark this answers is not in the document]
Comment (BrunoQuaresma) [reply site/src/components/SSHButton/SSHButton.tsx]: Done!

File: site/src/components/SSHButton/SSHButton.tsx (hunk 3 above)
Comment (BrunoQuaresma) [reply site/src/components/SSHButton/SSHButton.tsx:80]: Hm... I would add if the links are not explicit enough.

**Comment by ammario** [comment]
Great improvement @BrunoQuaresma

**Review by bpmct — COMMENTED** [review]
getting to this late, but this looks fantastic!

**Pull request merged**